docs/getting-started/asyncapi.md

```diff
@@ -0,0 +1,5 @@
+# AsyncAPI Support
+
+Spectral is a generic linter, but a lot of effort has been put in to making sure [AsyncAPI v2](https://www.asyncapi.com/docs/specifications/2.0.0/) is well supported.
+
+Run Spectral against a document without specifying a ruleset will trigger an auto-detect, where Spectral will look to see if `asyncapi: 2.0.0` is at the top of the file. If it finds it, it will load `spectral:asyncapi`, which is documented in our [Reference > AsyncAPI Rules](../reference/asyncapi-rules.md).
```

Comment:
> ...where Spectral will look to see if `asyncapi: 2.0.0` is at the top of the file

Is it looking for `asyncapi: 2.0.0` to be at the top of the file or the root of the YAML document? I'm asking because it may not be the first line.

Reply:
@fmvilas the engines searches for a node named `asyncapi` at the root of the document, then ensures it's valued with `2.0.0`.

@philsturgeon How should we reword that so that it's clearer?

Reply:
"
```suggestion
Run Spectral against a document without specifying a ruleset will trigger an auto-detect, where Spectral will look to see if `asyncapi: 2.0.0` in the root of the document. If it finds it, it will load `spectral:asyncapi`, which is documented in our [Reference > AsyncAPI Rules](../reference/asyncapi-rules.md).
```

Reply:
@philsturgeon @fmvilas Fixed